### ISSUE TITLE
Handle Failed Queue

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/door43.org#836]
## Story

As a *user* I want to see my job completed so that I don't wonder what happened.

As an *admin* I want to see the failed queue empty so that I can monitor it for system abnormalities.

## Notes

Based on the reason why a job is in the failed queue, we want to perform various actions on it.  Here is a start at specifying both:

* *Reason:* job is left over from previous testing  *Action:* Remove job.
* *Reason:* worker failed to complete job for unknown reason.  *Action:* Retry job and somehow flag it as a retry so that a subsequent fail can be detected.
* *Reason:* A retried job failed for an unknown reason.  *Action:* Log and remove job.

**Comment by RobH123** [comment]
Although I have left the system for now using the single rq _failed_ queue, I've modified the three queues (d43 webhook, callback, tx webhook) x2 (prodn/dev) to check and log only the stats for jobs from their own queues. So now the ...queue.length.failed gauges should now show the relevant figures (rather than all being the same as before).

I've also made each of the six queues above remove failed jobs from the failed queue after two weeks.

Retrying is not yet implemented.
